src/errors.rs

```diff
@@ -74,6 +76,7 @@ impl From<osrmc_sys::osrmc_error_t> for Error {
         let message = error.message().into_owned();
         let kind = match code.as_ref() {
             "NoRoute" => ErrorKind::NoRoute,
+            "InvalidValue" => ErrorKind::InvalidCoordinate,
             _ => ErrorKind::Osrmc { code, message }
         };
         Error { kind }
```

Comment:
This method converts from the strings that OSRM returns for each error (yuck!) into our `ErrorKind` enum.